src/pages/Register/SecondaryRegister.jsx

```diff
@@ -94,119 +119,125 @@ export default function SecondaryRegister() {
                 privacy policy
               </a>
             </p>
-            {isSuccess && (
+            {isSuccess && userData && (
               <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-                {({ dirty, values }) => (
-                  <Form className="flex flex-col gap-4">
-                    <div className="flex gap-4">
+                {({ dirty, values }) => {
+                  return (
+                    <Form className="flex flex-col gap-4">
+                      <div className="flex gap-4">
+                        <Field
+                          as="select"
+                          id="month"
+                          name="month"
+                          className={fieldClass + "w-20"}>
+                          <option value=""> Month</option>
+                          {generateSelectOptions(1, 12)}
+                        </Field>
+
+                        <Field
+                          as="select"
+                          id="day"
+                          name="day"
+                          className={fieldClass + "w-20"}>
+                          <option value=""> Day</option>
+                          {generateSelectOptions(1, 31)}
+                        </Field>
+
+                        <Field
```

Comment:
this is a birth date selector? if so, let the initial value be 1990, they can edit it after that, but it's a sane place to start, especially if there is scrolling